### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by Tufahel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+README.md
```
Comment (rotimiazeez) [inline .gitignore]: - [ ] Kindly remove the README.md from the gitignore and add test.md instead as stated in the requirement.😉

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
```
Comment (rotimiazeez) [inline README.md]: - [ ] Good job adding the README file but I will suggest you use the Microverse[ readme template](https://github.com/microverseinc/readme-template) and edit as necessary.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Tufahel 👋, 
## Good job so far!👍
### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — COMMENTED** [review]
Hi, @Tufahel  👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes

**Create a new pull requested following proper GitHub Flow as you have done earlier, add the link to this closed PR and request another review. I hope this makes some sense 😏  **




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**